.github/workflows/build_and_test.yml

```diff
@@ -75,33 +81,38 @@ jobs:
       - name: Build OSX Client
         run: npx gulp build --platform=osx
 
-  # TODO: setup fastlane for iOS
-  
-  # ios:
-  #   name: iOS
-  #   runs-on: macos-11
-  #   steps:
-  #     - name: Checkout
-  #       uses: actions/checkout@v2.3.4
+  ios:
+    name: iOS
+    runs-on: macos-11
+    timeout-minutes: 10
+    env:
+      MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Install Node
+        uses: actions/setup-node@v2.2.0
+        with:
+          node-version: 16
+          cache: npm
 
-  #     - name: Install Node
-  #       uses: actions/setup-node@v2.2.0
-  #       with:
-  #         node-version: 16
-  #         cache: npm
+      - name: Install NPM Dependencies
+        run: npm ci
 
-  #     - name: Install NPM Dependencies
-  #       run: npm ci
+      - name: Set XCode Version
+        run: sudo xcode-select -switch /Applications/Xcode_12.5.app
 
-  #     - name: Set XCode Version
-  #       run: sudo xcode-select -switch /Applications/Xcode_12.5.app
+      - name: Set Environment
+        run: echo "MATCH_GIT_BASIC_AUTHORIZATION=$(echo -n OutlineBot:${{ secrets.OUTLINE_CREDENTIALS_TOKEN }} | base64)" >> $GITHUB_ENV
```

Comment:
What's `GITHUB_ENV`?

Reply:
It's the env file that's loaded into each step of the job runner. I can't simply set this var and reference it later.

However, this is an artifact of when i was using `GITHUB_TOKEN`, which changes on every run. Now that this is stable i can put it in the `environment`

Reply:
also the variables didn't seem to populate - https://github.com/Jigsaw-Code/outline-client/runs/4446103793?check_suite_focus=true#step:6:299

secrets here:
![Screen Shot 2021-12-07 at 10 45 54 AM](https://user-images.githubusercontent.com/3759828/145060818-c2a3d0c0-887c-4abe-9055-7fadcc0591bd.png)

I wonder if i have to pass them in via `env` anyway